app/Decsys/Repositories/Contracts/IParticipantEventRepository.cs

```diff
@@ -13,19 +15,20 @@ interface IParticipantEventRepository
         string GetCollectionName(string participantId, LiteDatabase db);
         string GetCollectionName(int instanceId, string participantId);
         string GetParticipantId(int instanceId, string participantId);
-        IEnumerable<ParticipantEvent>_List(int instanceId, string participantId);
+        IEnumerable<Models.ParticipantEvent> _List(int instanceId, string participantId);
         string GetNextId(string participantId, int instanceId);
-        IEnumerable<ParticipantEvent> List(int instanceId, string participantId);
+        IEnumerable<Models.ParticipantEvent> List(int instanceId, string participantId);
         SurveyInstanceResults<ParticipantEvents> Results(int instanceId);
-        void Log(int instanceId, string participantId, ParticipantEvent e);
+        void Log(int instanceId, string participantId, Models.ParticipantEvent e);
 
-        ParticipantEvent Last(int instanceId, string participantId, string source, string type);
-        ParticipantEvent Last(int instanceId, string participantId, string type);
+        Models.ParticipantEvent Last(int instanceId, string participantId, string source, string type);
+        Models.ParticipantEvent Last(int instanceId, string participantId, string type);
         SurveyInstanceResults<ParticipantResultsSummary> ResultsSummary(int instanceId);
         List<string> GetAllParticipantLogs(int instanceId);
 
         ParticipantResultsSummary ResultsSummary(int instanceId, string participantId);
-        ParticipantResultsSummary ParticipantResultsSummary(SurveyInstance instance, string participantId);
+        ParticipantResultsSummary ParticipantResultsSummary(Data.Entities.SurveyInstance instance, string participantId);
```

Comment:
This is set as an entity in EventService on master, should they both be changed to take a model? Or simply the interface be changed?

Reply:
yeah they should both be changed to take a model.

That was probably a shortcut on master, that now needs changing because we're abstracting away from the db provider.